tests/test_server/test_lineage/test_column_lineage.py

```diff
@@ -0,0 +1,903 @@
+from http import HTTPStatus
+
+import pytest
+from httpx import AsyncClient
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from tests.fixtures.mocks import MockedUser
+from tests.test_server.utils.convert_to_json import (
+    datasets_to_json,
+    format_datetime,
+    inputs_to_json,
+    jobs_to_json,
+    operation_parents_to_json,
+    operations_to_json,
+    outputs_to_json,
+    run_parents_to_json,
+    runs_to_json,
+)
+from tests.test_server.utils.enrich import enrich_datasets, enrich_jobs, enrich_runs
+from tests.test_server.utils.lineage_result import LineageResult
+from tests.test_server.utils.merge import merge_io_by_jobs, merge_io_by_runs
+
+pytestmark = [pytest.mark.server, pytest.mark.asyncio, pytest.mark.lineage]
+
+
+async def test_get_column_lineage_by_dataset(
+    test_client: AsyncClient,
+    async_session: AsyncSession,
+    duplicated_lineage_with_column_lineage: LineageResult,
+    mocked_user: MockedUser,
+):
+    lineage = duplicated_lineage_with_column_lineage
+    dataset = lineage.datasets[0]
+
+    inputs = [input for input in lineage.inputs if input.dataset_id == dataset.id]
+    outputs = [output for output in lineage.outputs if output.dataset_id == dataset.id]
+
+    run_ids = {input.run_id for input in inputs} | {output.run_id for output in outputs}
+    runs = [run for run in lineage.runs if run.id in run_ids]
+    assert runs
+
+    job_ids = {run.job_id for run in runs}
+    jobs = [job for job in lineage.jobs if job.id in job_ids]
+    assert jobs
+
+    jobs = await enrich_jobs(jobs, async_session)
+    runs = await enrich_runs(runs, async_session)
+    [dataset] = await enrich_datasets([dataset], async_session)
+
+    since = min(run.created_at for run in runs)
+
+    response = await test_client.get(
+        "v1/datasets/lineage",
+        headers={"Authorization": f"Bearer {mocked_user.access_token}"},
+        params={
+            "since": since,
+            "start_node_id": dataset.id,
+            "include_column_lineage": True,
+        },
+    )
+
+    assert response.status_code == HTTPStatus.OK, response.json()
+    assert response.json() == {
+        "relations": {
+            "parents": run_parents_to_json(runs),
+            "symlinks": [],
+            "inputs": inputs_to_json(merge_io_by_runs(inputs), granularity="RUN"),
+            "outputs": outputs_to_json(merge_io_by_runs(outputs), granularity="RUN"),
+            "direct_column_lineage": [],
+            "indirect_column_lineage": [],
```

Comment:
Please add a note why column lineage here is empty. Either in test name or in docstring/comment